build.html

```
@@ -0,0 +1,7 @@
+<div data-fliplet-audio-player-id="{{id}}">
+  {{#if media.url}}
```

make sure this doesn't fail when `media` is undefined (e.g. cannot read property `url` or `undefined`; is handlebars affected by this?)

---

Since we don't use handlebars in strict mode, this shouldn't be an issue here. I guess they either have a mechanism for object safe navigation or just swallow the errors, either of which is fine.
https://handlebarsjs.com/reference.html
I think we can enable the strict mode, but we might enter a world of hell :)

---

@angelnikolov good finding, thanks for the reference! I like non-strict mode since it makes templates much more readable :)